Backend/api/views.py

```diff
@@ -94,14 +94,11 @@ def validate_date_format(requested_date):
 
 
 def send_pass_email(name, requested_date, invited_guests):
-  receiver_email = "n.novarlic@innopolis.ru" # this attribute has to be changed in future
+  receiver_email = "d.atonge@innopolis.university" # this attribute has to be changed in future
   
   visitors = "\t1. {name}\n".format(name=name)
-  count = 1
 
-  for guest in invited_guests:
-    count += 1
-    visitors += "\t{count}. {guest}\n".format(count=str(count), guest=guest)
+  visitors += "\t{guest}\n".format(guest=invited_guests)
```

Comment:
can you please use f string instead ? it's the way pep8 works

Reply:
This isn't my code bro, you had this codebase since Nikola left

Reply:
No problem it's because we introduced pep8 one week ago, I started replacing them but okay I can fix it later when I am fixing the other things